### PR TITLE
Issue6 7

### PR DESCRIPTION
**Pull request by MarkLoeser**
Added links to the get involved section text and fixed the issue with everything pulling to the right on small screens.

### CODE REVIEW

**Comment by TheJaredWilcurt** [comment]
:+1:

File: views/index-getInvolved.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
-<div class="getInvolvedSection container">
+<div class="getInvolvedSection">
   <h2>Get Involved</h2>
+  <div class="container">
 
   <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
```
Comment (aallbrig) [inline views/index-getInvolved.html]: If you are going to add a new `div`, you ought to "tab" over the inner content for legibility.  Right now you have...
```
<div class="container">
<div></div>
<div></div>
<div></div>
</div>
```
But you ought to tab it over by two spaces
```
<div class="container">
  <div></div>
  <div></div>
  <div></div>
</div>
```

Comment (TheJaredWilcurt) [inline views/index-getInvolved.html]: If you are using Sublime Text, you can highlight the lines, starting with the parent element, and then go to `Edit > Line > Reindent`.

**Comment by MarkLoeser** [comment]
I've fixed the merge conflicts and indented the divs in. (that sublime text tip worked!)

File: sass/_sponsors.scss [hunk 2]
```diff
@@ -1,8 +1,14 @@
 #sponsorSection {
+
+  background-color: #F2EFEF;
+  
+  .sponsors {
+  max-width: 960px;
```
Comment (TheJaredWilcurt) [inline sass/_sponsors.scss]: Lines 6-10 are not tabbed out correctlys

**Comment by MarkLoeser** [comment]
fixed the tabbing issue in the sponsors.scss file  - should be all set now

**Pull request merged**